MANUAL_REGRESSION.md

```diff
@@ -50,190 +53,209 @@ To check if a device&browser supports getUserMedia you can check this link - [ca
+
 1. Type valid mobile number connected to mobile test device and send
     - user should see `Check your mobile` screen
     - user should see option to resend link
 2. Click `Resend link`
-    - user should see `Continue your verification on mobile` screen again
+    - user should see `Continue verification on your mobile` screen again
     - user should be able to provide mobile number again
     - user should see the option to send SMS
 
-##### 7. Cross-device errors
+### 7. Cross-device errors
 (on private mode of yet another browser)
 
-0. Given user is on first page of cross-device flow
+Given user is on first page of cross-device flow
+
 1. Type invalid mobile number and click send
     - user should see an validation error
     - user should persist on the same screen
 2. **STAGING ONLY:** Type invalid US number (+1 234 567 8912) and click send
     - user should see `Something's gone wrong` error
     - user should persist on the same screen
 
-##### 8. Prevent opening cross-device URL on web browsers
+### 8. Prevent opening cross-device URL on web browsers
 (on private mode of: Google Chrome, Firefox, Safari and Microsoft Edge browsers)
 
-0. Given user is on first page of cross-device flow
+Given user is on first page of cross-device flow
+
 1. Click on the `Copy` button
 2. Open a new tab of the browser
 3. Paste the link
     - user should see `Something's gone wrong` message
     - user should see `You must open this link on a mobile device` message
     - user should see the icon with the phone, screen and the red cross
 
-####  9. Cross device transition between browsers with different liveness support
+###  9. Cross device transition between browsers with different liveness support
 (on private mode of: Google Chrome, Firefox)
 
-0. Given webcam is connected to the computer
+Given webcam is connected to the computer
+
 1. Open link with additional GET parameter `?liveness=true`
 2. Upload the ID documents in the browser
 3. On the liveness recording screen, wait for the `Use your mobile to continue face verification` message and select it
-4. Open the cross device link on a mobile device that doesn't have media recorder api support (Chrome on iOS)
+4. Open the cross device link on a mobile device that doesn't have media recorder API support (Chrome on iOS)
     - user should be taken to the selfie intro screen
-6. Open the cross device link on a mobile device that has media recorder api support (Chrome on Android)
+5. Open the cross device link on a mobile device that has media recorder API support (Chrome on Android)
     - user should be taken to the liveness intro screen
 
-0. Given webcam is not connected to the computer
+Given webcam is not connected to the computer
+
 1. Open link with additional GET parameter `?liveness=true`
 2. Upload the ID documents in the browser
-4. Open the cross device link on a mobile device that doesn't have media recorder api support (Chrome on iOS)
```

Comment:
`1...2...4...6` 🤔

Reply:
As discussed, have updated this list's numbers in my original commit on the assumption that there were probably steps that were removed but order number not updated when that happened.